CHANGELOG.rst

```diff
@@ -9,6 +9,12 @@ Added
 
 * Added possibility to add new values to the KV store via CLI without leaking them to the shell history. #5164
 
+* Replaced the static ``.socket`` sytemd units in deb and rpm packages with a python-based generator for the
+  ``st2api``, ``st2auth``, and ``st2stream`` services. The generators will get ``<ip>:<port>`` from ``st2.conf``
+  to create the ``.socket`` files dynamically. #5286 and st2-packages#706
+
+  Contributed by @nzlosh
```

Comment:
Apart of more technical details about the generators, perhaps adjusting a higher level note for the users like "stackstorm service ports are now configured in a single st2.conf place." would help others to understand the value behind this change.